app/scripts/modules/core/src/pipeline/config/pipelineConfigurer.js

```diff
@@ -24,13 +26,16 @@ module.exports = angular.module('spinnaker.core.pipeline.config.pipelineConfigur
         application: '=',
         plan: '<',
         isTemplatedPipeline: '<',
+        hasDynamicSource: '<',
+        templateError: '<',
       },
       controller: 'PipelineConfigurerCtrl as pipelineConfigurerCtrl',
       templateUrl: require('./pipelineConfigurer.html'),
     };
   })
+  .component('executionBuildTitle', react2angular(ExecutionBuildTitle, ['execution', 'defaultToTimestamp']))
```

Comment:
Let's keep this in a separate file. It feels hidden in here, tucked between a directive declaration and the directive's controller declaration.

Reply:
Fixed